src/GeniusYield/Test/Privnet/Ctx.hs

```diff
@@ -92,6 +89,19 @@ ctxNetworkId Ctx {ctxNetworkInfo} = GYPrivnet ctxNetworkInfo
 ctxUsers :: Ctx -> [User]
 ctxUsers ctx = ($ ctx) <$> [ctxUser2, ctxUser3, ctxUser4, ctxUser5, ctxUser6, ctxUser7, ctxUser8, ctxUser9]
 
+ctxWallets :: Ctx -> Wallets
```

Comment:
This whole testing wallets mechanism needs to be revamped and made better at some point. Distinction between funding wallets and normal wallets, variadic number of wallets etc.